quiche/src/recovery/congestion/bbr/per_ack.rs

```diff
@@ -258,13 +258,28 @@ fn bbr_check_drain(r: &mut Congestion, bytes_in_flight: usize, now: Instant) {
     }
 
     if r.bbr_state.state == BBRStateMachine::Drain &&
-        bytes_in_flight <= bbr_inflight(r, 1.0)
+        bbr_bytes_in_net(r, bytes_in_flight, now) <= bbr_inflight(r, 1.0)
     {
         // we estimate queue is drained
         bbr_enter_probe_bw(r, now);
     }
 }
 
+fn bbr_bytes_in_net(r: &Congestion, in_flight: usize, now: Instant) -> usize {
```

Comment:
Could you link to the [source](https://elixir.bootlin.com/linux/v6.13.7/source/net/ipv4/tcp_bbr.c#L437) and add a comment explaining the context

Reply:
Yeah, good point. Added a comment.